Tools/wpe/backends/fdo/WindowViewBackend.cpp

```diff
@@ -1063,7 +1063,20 @@ void WindowViewBackend::displayBuffer(struct wpe_fdo_egl_exported_image* image)
     struct wl_callback* callback = wl_surface_frame(m_surface);
     wl_callback_add_listener(callback, &s_frameListener, this);
 
+#if defined(ENABLE_BUFFER_DAMAGE_TRACKING) && ENABLE_BUFFER_DAMAGE_TRACKING
```

Comment:
Rather than a build option, I would prefer to use `epoxy_has_egl_extension()` or `GLContext::isExtensionSupported()` to check whether extensions `EGL_EXT_swap_buffers_with_damage` or `EGL_KHR_swap_buffers_with_damage` are available at run time.